cli/internal/libvirt/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM fedora@sha256:486fd5578f93fbc57a519e34ad4b7cac927c3f8a95409baedf0c19e9f287c207 AS deploy
```

Comment:
I think we should add a short README.md here with use and build instruction like with the qemu metadata docker image.